libs/model/src/aggregates/thread/GetThreads.query.ts

```diff
@@ -42,7 +42,7 @@ export function GetThreads(): Query<typeof schemas.GetThreads> {
         limit: _limit,
         offset: _limit * (_page - 1) || 0,
         from_date,
-        to_date: to_date || moment().toISOString(),
+        to_date: to_date || dayjs().toISOString(),
```

Comment:
Do we really need `dayjs` for this? What's wrong with just `new Date().toISOString()`

Reply:
not really, just from legacy find/replace but updated in last commit to new Date().toISOString() 👍